src/save_failed_screen.c

```diff
@@ -22,17 +22,11 @@
 
 #define CLOCK_WIN_TOP (MSG_WIN_TOP - 4)
 
-struct SaveFailedClockStruct
-{
-    bool16 clockRunning;
-    u16 timer; // appears to be unused, the only case its "used" cannot be reached normally
-    u8 unused[0xE]; // appears to be unused space. alternatively, there could have been multiple clock structs. however, neither of these cases are fulfilled, so the purpose of this space is unknown. one theory is that there is room for 3 more clock structs (2 16-bit values), so its possible GF intended there to be support for link-based save failed screens in case the synchronization failed, or it in fact was present in an earlier build but taken out for reasons.
-};
-
 extern u8 unk_2000000[];
 
-extern u16 gSaveFailedType;
-extern struct SaveFailedClockStruct gSaveFailedClockInfo;
+static EWRAM_DATA u16 gSaveFailedType = 0;
+static EWRAM_DATA u16 gSaveFailedClockInfo[9] = {0};
```

Comment:
It's not aligned to 4 bytes. Putting the gSaveFailedType variable into the struct didn't match either.